test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/function/IntToDoubleFunctionTest.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.function
+
+import org.junit.Assert._
+import org.junit.Test
+
+import java.util.function._
+
+class IntToDoubleFunctionTest {
+  @Test def testApply(): Unit = {
+    val f = new IntToDoubleFunction {
+      override def applyAsDouble(value: Int): Double = value.toDouble / 11d
+    }
+    assertEquals(f.applyAsDouble(3), 0.272, 0.001)
```

Comment:
Or just use the full result.

Reply:
I think epsilon is needed since JUnit deprecated `assertEquals(double expected, double actual)` and encourage to use `assertEquals(double expected, double actual, double delta)` .

Reply:
Yes but we can use a `0.0` value for the epsilon. We often do that.

Reply:
Fixed: https://github.com/scala-js/scala-js/pull/4365/commits/5122020b75961c7fe0a024337df0e8625a9a5999